app/src/main/java/ch/epfl/sdp/appart/login/FirebaseLoginService.java

```diff
@@ -18,12 +20,12 @@
 
 @Singleton
 public class FirebaseLoginService implements LoginService {
-    private final FirebaseAuth mAuth;
+    private FirebaseAuth mAuth;
 
     @Inject
     public FirebaseLoginService() {
         this.mAuth = FirebaseAuth.getInstance();
-        //mAuth.useEmulator("10.0.2.2", 9099);
+        this.mAuth.signOut(); //to unsure that no invalid user is cached

```

Comment:
At the creation of the service there shouldn't be any logged user.

Reply:
Oh i understand that the goal of this is to remove the default user. Is that right ?